deno.json

```diff
@@ -1,9 +1,17 @@
 {
+  "name": "@cmd-johnson/deno-oauth2-client",
+  "version": "1.0.3",
```

Comment:
Maybe this should be `2.0.0` as this PR makes the package no longer available via HTTP import specifiers.